### DISCUSSION TITLE
(1password) MSI installer

### DISCUSSION THREAD

**Discussion by gonzague** [chocolatey-community/chocolatey-packages#1821] [Ideas]
Hello

For information / I think it could help improve the package: 1Password exists as a MSI file.
You just have to go here : https://app-updates.agilebits.com/ , then spot the URL of the .exe file, for example https://c.1password.com/dist/1P/win6/1PasswordSetup-7.9.828.**exe**
and replace .exe by .msi https://c.1password.com/dist/1P/win6/1PasswordSetup-7.9.828.**msi** , you now have the MSI file :)

Would be nice to integrate this instead of the .exe ?

**Comment by AdmiringWorm** [comment]
Hmm, that is a good idea. Unfortunately since the MSI installer isn't linked to anywhere on their website (AFAIK), we can really use it.

For us to use an installer it needs to be:

1. Come from an official download location (exceptions can be made here for certain third-party sites, but that is rare).
2. It be linked to from a browsable location on their official site without guessing the name (changing URL from using exe, to msi in this case would be guessing).

**Reply by gonzague** [reply]
Hello @AdmiringWorm .. I did get the instructions from 1Password's support. I'll point them to this topic for point number 2.  I guess the rule number 1 is already okay haha .. doesnt get more official than the editor's website.

**Reply by AdmiringWorm** [reply]
Yeah, point 1 is okay. It was just included for completeness in case someone comes over this in the future.

**Reply by gonzague** [reply]
Alright I got the following reply:

> Thanks for reaching out to us. At this time, the MSI installer is only available to our Business and Teams customers and we don't have any plans to list it on our releases page.
> With that said, although I can't make any promises, I'll be sure to pass this along to the team to see if there's anything we can do.
